### DISCUSSION TITLE
Starttime of Medication

### DISCUSSION THREAD

**Discussion by izzypeazzy** [MIT-LCP/mimic-code#1822] [MIMIC-IV] [answered]
Hi everyone,
I am currently working on a sepsis related topic and hence with antibiotics in MIMIC IV. I am really grateful for the already existing antibiotics table in MIMIC IV derived but I have a question regarding the starttime. As far as I have seen, start- and endtime of mediacations are generally rounded to the hour (correct me if I am wrong) and I was wondering whether the actual time is ceiled or floored to the hour. 
Also, what exactly is the starttime? In the documentation it says "The start and stop times for the given prescribed medication" but there are no further details. As it is the only time mentioned in the antibiotics table I am assuming it is more important than e.g. the entertime. 
I appreciate any further information! Thank you!

**Comment by alistairewj** [comment, marked as answer]
The *antibiotics* table is derived from prescriptions, which is the intention to treat. So the start time and end times are approximate, more or less when the prescription is valid for. 

If you want exactly when the antibiotic was given, you'll have to switch to writing your own query using the *emar* and *emar_detail* tables. These have precise administrations (as the data is sourced from bar code scanning). Unfortunately not all patients have data in emar because the system was rolled out mid way through the data collection for MIMIC-IV. Roughly half of the patients will have data in the table. It's also a fairly dense table because it captures every single administration, e.g. if a dose requires 3 pills, there are 3 rows and then another "main" row describing the overall administration. There aren't too many queries available for it yet regrettably.

**Reply by izzypeazzy** [reply]
Alright, thank you for the explanation!
